django_auth_adfs/backend.py

```diff
@@ -122,6 +122,10 @@ def create_user(self, claims):
         # Create the user
         username_claim = settings.USERNAME_CLAIM
         usermodel = get_user_model()
+
+        if claims.get(username_claim, None) is None:
```

Comment:
Are we expecting the value to be truthy if found? If so you should be able to simplify to this:
```suggestion
        if not claims.get(username_claim):
```

Reply:
Yep, the name should be a string if found, so your suggestions looks fine to me, I usually use the "is None" solution when im not really sure of what kind of data should come, in this case the data should be always a string so I commit your suggestion.

Thank you :)